hedgehog/src/Hedgehog/Gen.hs

```diff
@@ -90,6 +90,7 @@ module Hedgehog.Gen (
   -- ** Combinations & Permutations
   , subsequence
   , shuffle
+  , shuffleSeq
```

Comment:
`shuffleSeq` can be part of the public `Hedgehog.Gen` API later if needed, e.g. if we get many use cases or requests for it. Because once we add it there, it's basically there forever.

For now let's keep it as part of the `Hedgehog.Internal.Gen` API as you've added it below. Apart from that, we can merge this and fine tune it later (e.g. remove the `#else` CPP branch as per the bench results shown earlier on).

Reply:
```suggestion
```